### DISCUSSION TITLE
How make attended transfer without second userAgent info?

### DISCUSSION THREAD

**Discussion by MikePozh** [sipsorcery-org/sipsorcery#1056] [Q&A]
Hello!

I'm developing my softphone and ran into a problem. It is necessary to transfer the call. To perform a transfer using the AttendedTransfer method, you need information on the SIPDialogue of the second agent, but I don’t have it. How can I transfer it? Maybe I can create a SIPDialogue object?

**Comment by sipsorcery** [comment]
There's an example of wiring up an attended transfer [here](https://github.com/sipsorcery-org/sipsorcery/tree/master/examples/SIPScenarios/AttendedTransferScenario).
